SymSettings.cfg

```diff
@@ -12,7 +12,7 @@ set GRID_Y 100                    # Height of world, just multiplied by width to
 set POP_SIZE -1                   # Starting size of the host population, -1 for full starting population
 set SYM_LIMIT 1                   # Number of symbiont allowed to infect a single host
 set START_MOI 1                   # Ratio of symbionts to hosts that experiment should start with
-set UPDATES 1001                  # Number of updates to run before quitting
+set UPDATES 2000                  # Number of updates to run before quitting
```

Comment:
Make sure not to add SymSettings so that the file in the repository isn't changed. You should revert this change in your fork

Reply:
This still needs to be fixed